src/main/java/org/jenkinsci/plugins/argusnotifier/BuildMetricFactory.java

```diff
@@ -95,8 +95,15 @@ private Metric getBuildTimeMetric(String labelForDisplayName, String metricStrin
         // TODO: metric.setNamespace(projectName);
         double timeInSeconds = (double) timeInMillis / 1000.0;
 
-        metric.setTags(TagFactory.buildStatusTags(jenkins,
+        String gitCommitId = jenkinsRunFormatter.getGitCommit();
```

Comment:
Can you please add a test case for this change?

Reply:
Yes, added one test case to cover gitCommit is a valid string and empty.

Reply:
Thank you!